bolt-modules/prompt/lib/puppet/functions/prompt.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'bolt/error'
+
+# Display a prompt and wait for a response.
+#
+# > **Note:** Not available in apply block
+Puppet::Functions.create_function(:prompt) do
+  # Wait until targets are available.
```

Comment:
Not quite.

Reply:
Heh, whoops. Definitely a leftover from a copy/paste. :P